gobusterdir/http.go

```diff
@@ -103,6 +108,11 @@ func (client *httpClient) makeRequest(fullURL, cookie string) (*int, *int64, err
 		return nil, nil, err
 	}
 
+	// disable HEAD requests if the target does not support the method
+	if opt.HeadRequests && (resp.StatusCode == 405 || resp.StatusCode == 501) {
```

Comment:
Is it worth noting this to the user somehow via logging/similar?

Reply:
Agreed here too. Definitely log something that indicates the fallback.

Reply:
Makes sense, I think this adds to improve the user experience.